.storybook/ssr/utils.js

```diff
@@ -46,6 +46,12 @@ const getPrerenderedContent = (webpackStats, params) => {
   return app.default(params);
 };
 
+const matchWebpackTest = (file, test) => (
+  test instanceof RegExp
+    ? test.test(file)
+    : test === file
```

Comment:
string? perhaps use .includes?

Reply:
Not sure if this line is actually needed at all. Almost all rules use a regex `test` (at least the ones we want to target here).

A string `test` may need a more complex check because file path could be relative or something like that? Not sure? 

I suggest: Always return false for strings and add a comment

Reply:
not sure myself, leave a comment yes